pkgs/desktops/budgie/budgie-gsettings-overrides/default.nix

```diff
@@ -31,9 +31,15 @@ let
     document-font-name="Noto Sans 10"
     monospace-font-name="Hack 10"
 
+    [org.gnome.desktop.peripherals.touchpad:Budgie]
+    tap-to-click=true
+
     [org.gnome.desktop.wm.preferences:Budgie]
     titlebar-font="Noto Sans Bold 10"
 
+    [org.gnome.mutter:Budgie]
+    edge-tiling=true
+
```

Comment:
Does this work? Or do we need to re-add magpie to list of override packages for it :joy:

Reply:
I did the change on my laptop but forgot to check if it worked on the VM :facepalm: